moped-database/migrations/1635521196468_create_table_public_moped_project_types/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE "public"."moped_project_types"("project_id" integer NOT NULL, "type_name" text NOT NULL, "project_type_id" integer NOT NULL, "date_added" timestamptz DEFAULT clock_timestamp(), "added_by" integer, PRIMARY KEY ("project_type_id") , FOREIGN KEY ("project_type_id") REFERENCES "public"."moped_types"("type_id") ON UPDATE restrict ON DELETE restrict, FOREIGN KEY ("project_id") REFERENCES "public"."moped_project"("project_id") ON UPDATE restrict ON DELETE restrict, UNIQUE ("project_type_id"));
```

Comment:
For this table - my understanding is that it's a table that maps `moped_project` and `moped_types` in a many:many relationship.

I took the SQL above and ran it through a formatter for readability:

```
CREATE TABLE "public"."moped_project_types"(
    "project_id" integer NOT NULL,
    "type_name" text NOT NULL,
    "project_type_id" integer NOT NULL,
    "date_added" timestamptz DEFAULT clock_timestamp(),
    "added_by" integer,
    PRIMARY KEY ("project_type_id"),
    FOREIGN KEY ("project_type_id") REFERENCES "public"."moped_types"("type_id") ON UPDATE restrict ON DELETE restrict,
    FOREIGN KEY ("project_id") REFERENCES "public"."moped_project"("project_id") ON UPDATE restrict ON DELETE restrict,
    UNIQUE ("project_type_id")
);
```

There is a `text` column defined in this table called `type_name` which I think may be vestigial. The actual textual representation of a type is contained in `moped_types.type_name` and is referenced in this table by a FK ID. I think keeping this text column around isn't intended. @zappfyllc - do you think that's the case?

If so, I suggest we exclude that column in this `CREATE` statement. Keeping it around won't hurt anything, but it may be confusing to developers implementing code which uses these tables.

Reply:
Yep, we can definitely omit that column @frankhereford!